docs/angular-integration.asciidoc

```diff
@@ -3,6 +3,11 @@
 
 This document covers how to use Real User Monitoring JavaScript agent with Angular applications.
 
+[[angular-supported-versions]]
+==== Supported versions
+
+This integration supports Angular versions ≤ 8.2.2.
```

Comment:
We can call it as <9. We just test our package on 8.2.2 locally for E2E tests but all minor versions should work.